build_deploy.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# AppSRE team CD
+
+set -exv
+
+BASE_IMG="hive-controller"
+IMG="${BASE_IMG}:latest"
```

Comment:
Will we expect these to always publish to "latest" or should it use a git version here? Actually it looks like below we get a tag for both? That is cool.

Reply:
they, that is correct, latest is a rolling tag, but you also get the short git commit as the tag so you can pin it